src/App.js

```diff
@@ -37,7 +37,8 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={ tweetsArray[0] }>
+      </Tweet>
```

Comment:
This can be a self closing tag - shorter syntax.

`<Tweet tweet={ tweetsArray[0] }/>`